### ISSUE TITLE
Configuration file

### ISSUE THREAD

**Issue by Arritmic** [phuselab/pyVHR#19]
Hi,

Could you confirm that the configuration file you have used to obtain the results in the paper is the _default_test.cfg file_?
If not, could you share the exact file you have used to do the evaluation in the paper?

Thanks in advance,
Tino A

**Comment by aledamelio** [comment]
Hi @Arritmic,

the configuration file used for the experiments in the paper has minor differences with respect to the default_test.cfg. In particular, you should change the following variables:

winsize = 10
winsizeGT = 7
ROImask = skin_adapt

However, we are currently working on an updated version of the package with new functionalities and experiments, so... stay tuned!

Thank you for your interest,
Alessandro

**Comment by Arritmic** [comment]
Okay, thanks Alessandro.
I will be very interested on those changes to perform our evaluation.

By the way, I am not sure if you had time to check the last _pull request_ I have made, but I think I have fixed an issue that it is not super critical according the results I have obtained, but it would be more coherent. Or also it is possible that I had not understood well the idea.

Cheers and thanks for the answer.
Tino A.
